npm/private/npm_package_store.bzl

```diff
@@ -360,30 +375,30 @@ deps of npm_package_store must be in the same package.""" % (ctx.label.package,
         # closure of all the entire package store deps, we can safely add just `files` from each of
         # these to `transitive_files_depset`; doing so reduces the size of `transitive_files_depset`
         # significantly and reduces analysis time and Bazel memory usage during analysis
+        sources_depset = depset(transitive = [jsinfo.sources for jsinfo in js_infos])
+        types_depset = depset(transitive = [jsinfo.types for jsinfo in js_infos])
         for npm_package_store_info in npm_package_store_infos:
             transitive_files_depsets.append(npm_package_store_info.files)
 
-    transitive_files_depset = depset(files, transitive = transitive_files_depsets)
-
-    files_depset = depset(files)
+    npm_sources = depset(files, transitive = [jsinfo.npm_sources for jsinfo in js_infos])
 
     providers = [
         js_info(
             target = ctx.label,
-            npm_sources = transitive_files_depset,
-        ),
-        DefaultInfo(
```

Comment:
Why was `DefaultInfo` here? Or maybe the question is why are tests still passing without it?

Reply:
I don't think it is load bearing in any test cases and likely not in any use cases in the wild so we can remote it... probably just added so that if you `bazel build` the target you get some default output and if you depend on it directly in some binary rule that doesn't understand `js_info` then things still work. Safe to remove IMO.